src/pages/user/questions/create.tsx

```diff
@@ -18,23 +18,24 @@ import { TextArea } from "../../../components/Form/TextArea";
 import Header from "../../../components/Header";
 
 import * as yup from "yup";
-import {
-  QuestionCategory,
-  QuestionPayload,
-} from "../../../types/question";
+import { QuestionPayload } from "../../../types/question";
 import api from "../../../services/api";
 import { useRouter } from "next/router";
 import Head from "next/head";
 import Sidebar from "../../../components/Sidebar";
+import { findCategoriesAtFirestore } from "../../../services/firebase/firestore/read";
+import { InferGetServerSidePropsType } from "next";
 
 const formCreateSchema = yup.object().shape({
   title: yup.string().required(),
   description: yup.string().required(),
-  category: yup.string().oneOf(Object.values(QuestionCategory)).required(),
+  category: yup.string().required(),
   complexity: yup.number().min(0).max(5).required(),
 });
 
-export default function CreateUser() {
+export default function CreateUser({

```

Comment:
Está correto o nome dessa function?
```suggestion
export default function CreateQuestion({
```

Reply:
Bem observado Vitor.